openassessment/xblock/static/sass/oa/utilities/_developer.scss

```diff
@@ -163,7 +159,7 @@ $link-hover: $edx-blue-l1 !default; // from our Pattern Library http://ux.edx.or
   .student-training__display__title {
     @extend %t-heading;
     margin-bottom: ($baseline-v/2);
-    color: $heading-secondary-color;
```

Comment:
We are losing the subtlety of different heading levels having slightly different colors, but now they are at least accessible.